doc/maintenance.rst

```diff
@@ -0,0 +1,76 @@
+Maintenance
+===========
+
+This work is supported and maintained by `Anaconda
+<https://www.anaconda.com>`_.  PyViz Topics is part of the `PyViz
+<https://pyviz.org>`_ project to showcase Python data visualization tools.
+The primary PyViz maintainers are Philipp Rudiger, James A. Bednar,
+Julia Signell, and Jean-Luc Stevens, with bug reports and patches from
+numerous members of the Github community.
+
+Website Maintenance
+===================
+The website building occurs in travis jobs that are triggered by commit messages.
+
+**NOTE:** When you merge PRs, you should be very careful about what is included
+in the commit messages.
```

Comment:
That sounds scary; can you explain what to watch out for?

Reply:
Oh just that you aren't triggering unnecessary builds. Maybe I should take out the "very".

Reply:
Just say something to explain what you mean...